docs/services/add-service.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Add a service
+intro: Add services to servers and applications so they can easily be managed in one place.
```

Comment:
@pvdevs Suggestion: perhaps emphasize `Linux service` (easier to be Google-d when user looks for help :D) and also highlight the fact that services can be linked to specific `servers` and `applications`?

```suggestion
intro: Add Linux services to your Devopness environments, so they can be easily managed and linked to all servers and applications that depend on them.
```